flink-streaming-java/src/test/java/org/apache/flink/streaming/api/StreamExecutionEnvironmentTest.java

```diff
@@ -384,70 +364,72 @@ public void testUserDefinedJobNameWithConfigure() {
         config.set(ExecutionOptions.BUFFER_TIMEOUT_ENABLED, true);
         testBufferTimeout(config, env);
     }
 
     @Test
-    public void testBufferTimeoutDisabled() {
+    void testBufferTimeoutDisabled() {
         Configuration config = new Configuration();
         config.set(ExecutionOptions.BUFFER_TIMEOUT_ENABLED, false);
         StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
 
         // The execution.buffer-timeout's default value 100ms will not take effect.
         env.configure(config, this.getClass().getClassLoader());
-        assertEquals(ExecutionOptions.DISABLED_NETWORK_BUFFER_TIMEOUT, env.getBufferTimeout());
+        assertThat(env.getBufferTimeout())
+                .isEqualTo(ExecutionOptions.DISABLED_NETWORK_BUFFER_TIMEOUT);
 
         // Setting execution.buffer-timeout's to 0ms will not take effect.
-        config.setString(ExecutionOptions.BUFFER_TIMEOUT_INTERVAL.key(), "0ms");
+        config.setString(ExecutionOptions.BUFFER_TIMEOUT.key(), "0ms");
         env.configure(config, this.getClass().getClassLoader());
-        assertEquals(ExecutionOptions.DISABLED_NETWORK_BUFFER_TIMEOUT, env.getBufferTimeout());
+        assertThat(env.getBufferTimeout())
+                .isEqualTo(ExecutionOptions.DISABLED_NETWORK_BUFFER_TIMEOUT);
 
         // Setting execution.buffer-timeout's to -1ms will not take effect.
-        config.setString(ExecutionOptions.BUFFER_TIMEOUT_INTERVAL.key(), "-1ms");
+        config.setString(ExecutionOptions.BUFFER_TIMEOUT.key(), "-1ms");
         env.configure(config, this.getClass().getClassLoader());
-        assertEquals(ExecutionOptions.DISABLED_NETWORK_BUFFER_TIMEOUT, env.getBufferTimeout());
+        assertThat(env.getBufferTimeout())
+                .isEqualTo(ExecutionOptions.DISABLED_NETWORK_BUFFER_TIMEOUT);
     }
 
     private void testBufferTimeout(Configuration config, StreamExecutionEnvironment env) {
         env.configure(config, this.getClass().getClassLoader());
-        assertEquals(
-                ExecutionOptions.BUFFER_TIMEOUT_INTERVAL.defaultValue().toMillis(),
-                env.getBufferTimeout());
+        assertThat(env.getBufferTimeout())
+                .isEqualTo(ExecutionOptions.BUFFER_TIMEOUT.defaultValue().toMillis());
 
-        config.setString(ExecutionOptions.BUFFER_TIMEOUT_INTERVAL.key(), "0ms");
+        config.setString(ExecutionOptions.BUFFER_TIMEOUT.key(), "0ms");
         env.configure(config, this.getClass().getClassLoader());
-        assertEquals(0, env.getBufferTimeout());
+        assertThat(env.getBufferTimeout()).isZero();
 
         try {
-            config.setString(ExecutionOptions.BUFFER_TIMEOUT_INTERVAL.key(), "-1ms");
+            config.setString(ExecutionOptions.BUFFER_TIMEOUT.key(), "-1ms");
             env.configure(config, this.getClass().getClassLoader());
             fail("exception expected");
         } catch (IllegalArgumentException ignored) {
```

Comment:
Maybe we can use `assertThatThrowBy` instead of this `try-catch` block.

Reply:
Good catch, updated it and another one.